### PR TITLE
COSMIT PEP-8-ted ``pymc3.models``

### PR DESCRIPTION
**Pull request by superbobry**
As discussed in 7c04dfee79d4a626d738638834af2aad681c925b. 

The code and docstrings still require some polishing though.

### CODE REVIEW

File: pymc3/model.py [hunk 1]
```diff
@@ -26,47 +25,45 @@ def __init__(self, obj, method_name):
...
-                        default=None,
-                        wrapper=None,
-                        override=False):
+
+def incorporate_methods(source, destination, methods, default=None,
+                        wrapper=None, override=False):
     """
     Add attributes to a destination object which points to
-    methods from from a source object.
+    methods from from a source objecT.
```
Comment (twiecki) [inline pymc3/model.py]: typo

File: pymc3/model.py [hunk 2]
```diff
@@ -89,8 +87,11 @@ def _get_named_nodes(graph, nodes):
             nodes.update(_get_named_nodes(i, nodes))
     return nodes
 
+
 class Context(object):
-    """Functionality for objects that put themselves in a context using the `with` statement."""
+    """Functionality for objects that put themselves in a context using
+    the `with` statemenT.
```
Comment (twiecki) [inline pymc3/model.py]: typo

File: pymc3/model.py [hunk 3]
```diff
@@ -218,34 +229,39 @@ def cont_vars(self):
...
                 self.free_RVs.append(var)
             else:
-                var = TransformedRV(name=name, distribution=dist, model=self, transform=dist.transform)
+                var = TransformedRV(name=name, distribution=dist, model=self,
+                                    transform=disT.transform)
                 if self.verbose:
                     print('Applied {transform}-transform to {name}'
                           ' and added transformed {orig_name} to model.'.format(
-                              transform=dist.transform.name,
+                              transform=disT.transform.name,
```
Comment (twiecki) [inline pymc3/model.py]: Overly aggressive replace-string ;)

File: pymc3/model.py [hunk 4]
```diff
@@ -218,34 +229,39 @@ def cont_vars(self):
...
                               name=name,
-                              orig_name='{}_{}'.format(name, dist.transform.name)))
+                              orig_name='{}_{}'.format(name, disT.transform.name)))
```
Comment (twiecki) [inline pymc3/model.py]: mark

File: pymc3/model.py [hunk 5]
```diff
@@ -303,12 +320,13 @@ def fn(self, outs, mode=None, *args, **kwargs):
...
-        Compiled Theano function"""
+        Compiled Theano function
+        """
         return LoosePointFunc(self.makefn(outs, mode, *args, **kwargs), self)
 
     def fastfn(self, outs, mode=None, *args, **kwargs):
-        """Compiles a Theano function which returns `outs` and takes values of model
-        vars as a dict as an argument.
+        """Compiles a Theano function which returns `outs` and takes values
+        of model vars as a dict as an argumenT.
```
Comment (twiecki) [inline pymc3/model.py]: mark

File: pymc3/model.py [hunk 6]
```diff
@@ -317,13 +335,14 @@ def fastfn(self, outs, mode=None, *args, **kwargs):
...
+        Compiled Theano function as point function.
+        """
         f = self.makefn(outs, mode, *args, **kwargs)
         return FastPointFunc(f)
 
     def profile(self, outs, n=1000, point=None, profile=True, *args, **kwargs):
-        """Compiles and profiles a Theano function which returns `outs` and takes values of model
-        vars as a dict as an argument.
+        """Compiles and profiles a Theano function which returns `outs` and
+        takes values of model vars as a dict as an argumenT.
```
Comment (twiecki) [inline pymc3/model.py]: mark

File: pymc3/model.py [hunk 7]
```diff
@@ -377,33 +398,32 @@ def fastfn(outs, mode=None, model=None):
...
-    Compiled Theano function as point function."""
+    Compiled Theano function as point function.
+    """
     model = modelcontext(model)
     return model.fastfn(outs,mode)
 
 
 def Point(*args, **kwargs):
-    """Build a point. Uses same args as dict() does.
+    """Build a poinT. Uses same args as dict() does.
```
Comment (twiecki) [inline pymc3/model.py]: mark

File: pymc3/model.py [hunk 8]
```diff
@@ -362,13 +381,15 @@ def fn(outs, mode=None, model=None, *args, **kwargs):
...
+    Compiled Theano function
+    """
     model = modelcontext(model)
     return model.fn(outs,mode, *args, **kwargs)
 
+
 def fastfn(outs, mode=None, model=None):
     """Compiles a Theano function which returns `outs` and takes values of model
-    vars as a dict as an argument.
+    vars as a dict as an argumenT.
```
Comment (twiecki) [inline pymc3/model.py]: mark

File: pymc3/model.py (hunk 3 above)
Comment (superbobry) [inline pymc3/model.py]: Bah, sorry. I though I should double check after doing the replace and completely forgot about it :)

**Comment by twiecki** [comment]
Thanks @superbobry!

**Pull request merged**